e2e/e2etests/e2etests.go

```diff
@@ -489,6 +489,7 @@ var AllE2ETests = []runner.E2ETest{
 		"withdraw ERC20 from ZEVM and authenticated call a contract",
 		[]runner.ArgDefinition{
 			{Description: "amount", DefaultValue: "1000"},
+			{Description: "gas limit for withdraw and call", DefaultValue: "250000"},
```

Comment:
I'm thinking of using 100000 instead by default as it is what we use on most chain so we replicate better live network case
Only Arbitrum need a higher gas limit?

Reply:
commit https://github.com/zeta-chain/node/pull/3972/commits/bde4ad13f7abc18d4189b8df654088912b6f570f.
CI erc20 withdrawAndCall failed on Arbitrum and Avalanche.